pkg/util/resources.go

```diff
@@ -40,6 +40,21 @@ func ParseAPIGroupsString(apiGroups string) sets.Set[string] {
 	return groupsSet
 }
 
+func ParseControllersString(controllers string) sets.Set[string] {
+	if controllers == "*" {
```

Comment:
`controllers == ""` will produce an equivalent result (i.e., treated the same by ShouldStartController), right? If so then I do not see the need to support two ways to say the same thing.

Reply:
Yes, removed `*` related code.

Reply:
Sorry, I should have also pointed out that I am not a fan of irregularity.  When absence of value X from set S means one thing, except it means the opposite when set S is empty, that is irregular. I am not going to insist on this, but it is FYI.

Reply:
Right. So I am OK with having "*" mean "every". My real complaint is having empty set mean full set.

Reply:
Not a blocker.